models/tracker/src/migration.ts

```diff
@@ -504,6 +504,35 @@ async function restoreTaskTypes (client: MigrationClient): Promise<void> {
   }
 }
 
+async function migrateIssueStatuses (client: MigrationClient): Promise<void> {
+  await client.update(
+    DOMAIN_TX,
+    {
+      objectClass: task.class.TaskType,
+      'attributes.ofClass': tracker.class.Issue,
+      'attributes.statusClass': core.class.Status
+    },
+    {
+      $set: {
+        'attributes.statusClass': tracker.class.IssueStatus
+      }
+    }
+  )
+
+  await client.update(
+    DOMAIN_STATUS,
```

Comment:
Statuses are in the model now. `DOMAIN_STATUS` was kept just in case if migration goes wrong. We'll drop it soon.

Reply:
It is 239 before statuses are in model actually.

Reply:
Ah, ok. Should be fine then.